salt/states/service.py

```diff
@@ -966,3 +966,40 @@ def mod_watch(name,
     ret['comment'] = 'Service {0}'.format(past_participle) if result else \
                      'Failed to {0} the service'.format(verb)
     return ret
+
+
+def systemctl_reloaded(name):
+    '''
+    Ensure that the systemctl daemon is reloaded.
+
+    .. note::
+        This state is only available on minions which use systemd.
+
+    name
+        Not used, the state_ID.
+
+    .. code-block:: yaml
+
+        systemd_daemon_is_reloaded:
+          service.systemctl_reloaded
+
+    '''
+    ret = {'name': name,
+           'changes': {},
+           'result': True,
+           'comment': 'Systemd is reloaded'}
+
+    if 'service.systemctl_reload' not in __salt__:
+        ret['result'] = False
+        ret['comment'] = "'systemctl daemon_reload' modules not available on this minion."
+    elif __opts__.get('test', False):
+        ret['result'] = None
+        ret['comment'] = 'Systemd would have been reloaded'
+    else:
+        try:
+            __salt__['service.systemctl_reload']()
```

Comment:
Does the execution module report status of the reload operation? If so, you should return that rather than hardcoding True

Reply:
No, the module `service.systemctl_reload` return always True. 

And the commande on linux : `systemctl daemon-reload` did not return status of the operation.